app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -53,7 +53,7 @@ def update
   # reorder FAQs
   def update_positions
     if params[:wrangling_guidelines]
-      @wrangling_guidelines = WranglingGuideline.reorder(params[:wrangling_guidelines])
+      @wrangling_guidelines = WranglingGuideline.reorder(params[:wrangling_guidelines].map { |s| s.to_i.to_s })
```

Comment:
Should we maybe use `g` as our variable here, as in `guideline`? I'm not sure what the `s` stands for.

Reply:
s is for string :)

Reply:
Will be g in the next push

Reply:
I think this can just have the method renamed in the model, like ticking suggested, so a variable rename won't be necessary

https://github.com/otwcode/otwarchive/blob/3862fc69353d3c9c6e06550ac860b6aa067170e5/app/models/wrangling_guideline.rb#L9-L11